streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractKeyValueStoreTest.java

```diff
@@ -67,6 +69,34 @@ public void after() {
         return result;
     }
 
+    @Test
+    public void testDelete() {
```

Comment:
nit: `shouldDeleteIfSerializedValueIsNull`

Reply:
It actually test more than that: both direct `delete` and `put with null bytes`.

Reply:
Split into two test? Or rename accordingly.`testDelete` is not a good name IMHO.